ethereum/eip712/eip712.go

```diff
@@ -363,6 +355,55 @@ func jsonNameFromTag(tag reflect.StructTag) string {
 	return parts[0]
 }
 
+func unpackIfAny(
+	cdc codectypes.AnyUnpacker,
+	_fieldType reflect.Type,
+	_field reflect.Value,
```

Comment:
the function is already private so we don't need to use `_` to define priv vars

```suggestion
	fieldType reflect.Type,
	field reflect.Value,
```